opae.spec

```diff
@@ -96,53 +126,53 @@ echo "" > %{_sysconfdir}/ld.so.conf.d/opae-c.conf
 ldconfig
 
 %postun
+ldconfig
 
 %pre
 
 %preun
 rm -f -- %{_sysconfdir}/ld.so.conf.d/opae-c.conf 
-ldconfig
 
 %files
 %defattr(-,root,root,-)
-
-%files libs
-%defattr(-,root,root,-)
+%dir %{_datadir}/opae
+%doc %{_datadir}/opae/RELEASE_NOTES.md
 %{_libdir}/libopae-c.so*
-%{_libdir}/libopae-cxx*
-%{_libdir}/libxfpga.so*
+%{_libdir}/libopae-cxx-core.so*
+%dir %{_libdir}/opae
+%{_libdir}/opae/libxfpga.so*
 %{_libdir}/libbmc.so*
-%{_libdir}/libmodbmc.so*
+%{_libdir}/opae/libmodbmc.so*
 %{_libdir}/libbitstream.so*
 %{_libdir}/libboard_rc.so*
 %{_libdir}/libboard_vc.so*
 
 %files devel
 %defattr(-,root,root,-)
-%{_bindir}/hello_fpga
 %dir %{_includedir}/opae
 %{_includedir}/opae/*
 %dir %{_includedir}/safe_string
 %{_includedir}/safe_string/safe_string.h
 %{_libdir}/libsafestr.a
-%dir %{_datadir}/opae
-%dir %{_datadir}/opae/*
-%{_datadir}/opae/*
 %dir %{_usr}/src/opae
-%{_usr}/src/opae/*
+%{_usr}/src/opae/samples/hello_fpga.c
+%{_usr}/src/opae/samples/hello_events.c
+%{_usr}/src/opae/samples/object_api.c
+%{_usr}/src/opae/cmake/modules/*
```

Comment:
I think the FindOPAE.cmake modules might be better in another installation directory:
```
<prefix>/lib/opae-1.4/opae-config.cmake
<prefix>/lib/opae-1.4/opae-config-version.cmake
```

https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/Packaging#package-version-files

Reply:
I see.  Any objection if we just omit it for now?  It's not really critical for this effort.